test-dhcp/003-teardown.bats

```diff
@@ -5,6 +5,13 @@
 
 load helpers
 
+# Redefine start_proxy locally to enable debug logging for this test only.
+# This version will be used instead of the one from helpers.bash.
+function start_proxy() {
+  RUST_LOG=debug ip netns exec "$NS_NAME" $NETAVARK dhcp-proxy --dir "$TMP_TESTDIR" --uds "$TMP_TESTDIR" &>"$TMP_TESTDIR/proxy.log" &
+  PROXY_PID=$!
+}
```

Comment:
I see no reason why we should not have debug log tunred on for all tests. I rather not duplicate this here.